diff --git a/contrib/plugins/meson.build b/contrib/plugins/meson.build
index 1876bc7..d26deb1 100644
--- a/contrib/plugins/meson.build
+++ b/contrib/plugins/meson.build
@@ -1,5 +1,6 @@
 contrib_plugins = ['bbv', 'cache', 'cflow', 'drcov', 'execlog', 'hotblocks',
-                   'hotpages', 'howvec', 'hwprofile', 'ips', 'stoptrigger']
+                   'hotpages', 'howvec', 'hwprofile', 'ips', 'mmio_profile',
+                   'stoptrigger']
 if host_os != 'windows'
   # lockstep uses socket.h
   contrib_plugins += 'lockstep'
